docs/source/building.rst

```diff
@@ -303,7 +303,7 @@ Regression Manager
 
     Enable to collect Python coverage data for user code.
     For some simulators, this will also report :term:`HDL` coverage.
-    If :envvar:`COCOTB_COVERAGE_RCFILE` is not set, branch coverage is collected
+    If :envvar:`COVERAGE_RCFILE` is not set, branch coverage is collected
```

Comment:
Rebase now that #4891 is in?

Reply:
They were separate commits, but yeah it's rebased now.